### PR TITLE
Document new features brought by PR #831

### PR DESCRIPTION
**Pull request by bellingard**
@BanzaiMan Here is the PR to update the page of the SonarQube add-on and document the latest features brought by https://github.com/travis-ci/travis-build/pull/831 - which BTW work beautifully :-)

@plaindocs, I guess it's up to you to review it - feel free to come back to me if you've got any question!

Thanks!

### CODE REVIEW

**Comment by bellingard** [comment]
FYI @dbmeneses

File: user/sonarqube.md [hunk 1]
```diff
@@ -12,42 +12,87 @@ Please refer to the [SonarQube documentation](http://redirect.sonarsource.com/do
...
 
 ## Upcoming improvements
 
 Next versions of this add-on will provide the following features:
 
-- Remove the need to pass the `sonar.login` parameter in the command-line
-- Proper and transparent management of branches (currently, you will have to use the `sonar.branch` parameter to properly analyze different branches of your repository)
-- Less configuration and scripting required to analyze pull requests
+- No need to define a third-party GitHub user for pull request analysis
+- Support for external pull requests
```
Comment (plaindocs) [inline user/sonarqube.md]: We usually punctuate list. :-)

File: user/sonarqube.md [hunk 2]
```diff
@@ -12,42 +12,87 @@ Please refer to the [SonarQube documentation](http://redirect.sonarsource.com/do
...
   # the following command line builds the project, runs the tests with coverage and then execute the SonarQube analysis
-  - mvn clean org.jacoco:jacoco-maven-plugin:prepare-agent install sonar:sonar -Dsonar.login=$SONAR_TOKEN
+  - mvn clean org.jacoco:jacoco-maven-plugin:prepare-agent install sonar:sonar
+```
+
+Please take a look at [the live Maven-based example project](https://github.com/SonarSource/sq-com_example_java-maven-travis) to know more about this use case.
+
+## Activation for branches
+
+By default, the SonarQube.com add-on will analyze and push results only for the master branch. You can activate it on branches thanks to the `branches` parameter:
```
Comment (plaindocs) [inline user/sonarqube.md]: > By default, the SonarQube.com add-on only analyzes the master branch. Activate it on other branches by specifying them in the `branches` parameter:

something like?

File: user/sonarqube.md [hunk 3]
```diff
@@ -12,42 +12,87 @@ Please refer to the [SonarQube documentation](http://redirect.sonarsource.com/do
...
+  sonarqube:
+    token:
+      secure: ********* # encrypted value of your token
 script:
   # other script steps might be done before running the actual SonarQube analysis
-  - sonar-scanner -Dsonar.login=$SONAR_TOKEN
+  - sonar-scanner
 ```
 
+Please take a look at [the live example project](https://github.com/SonarSource/sq-com_example_standard-sqscanner-travis) to know more about this standard use case.
```
Comment (plaindocs) [inline user/sonarqube.md]: Sorry to be picky, maybe move the "the" out of the link text

**Comment by plaindocs** [comment]
Couple of fairly picky comments, otherwise looks good to me.

**Comment by bellingard** [comment]
Thanks @plaindocs! I've applied all of your feedback.

@BanzaiMan, looks like this is ready to get published! Thanks.

**Comment by bellingard** [comment]
_(Ooooops, sorry, I pressed the button on the wrong tab of my browser... @BanzaiMan , this is still valid and ready to be pushed to production!)_

**Comment by bellingard** [comment]
FYI @BanzaiMan , I've added the requirements about the environment to prevent issues like https://github.com/travis-ci/travis-ci/issues/6631.

**Pull request merged**